### ISSUE TITLE
Support for private models

### ISSUE THREAD

**Issue by phantomas1234** [biosustain/memote#25]
Is it possible to password protect gh-pages websites?

**Comment by Midnighter** [comment]
I'm not aware of any functionality for that. If you want to keep things private you have to run everything locally and share the report HTML file yourself? Local workflows already work well so I don't feel the need to go beyond that. If you do find a way to have gh-pages with authentication I'm happy to reconsider.

I suppose an option could be to use a different branch that is not automatically published. You create a private repository and then just have an `index.html` lying in a particular branch. Anyone with access to the repo can look at that.

**Comment by phantomas1234** [comment]
Second option would already be ok. First option, wouldn't something simple as .htaccess already do the trick? I know that you can keep a gh-pages branch private.

**Comment by Midnighter** [comment]
How would `.htaccess` do authentication?

According to this [SO answer](http://stackoverflow.com/a/19058071) (which is old, I know) there's a Jekyll plugin that helps. For now I'd point users to the second option or they are welcome to come up with a solution themselves, unless you say it's really high priority for us.

**Comment by phantomas1234** [comment]
No, created this issue mainly as a reminder that most people wouldn't be comfortable sharing models in construction.

**Comment by Midnighter** [comment]
Once we fully support Gitlab and the Gitlab runner this will be supported.
